src/nestjs-back/src/games/class/Constants.ts

```diff
@@ -26,6 +26,7 @@ export enum GameState {
 
 export enum userStatus {
```

Comment:
A type should start with uppercase `UserStatus`

Reply:
Thank you for the review! It has been renamed: [8c765ed](https://github.com/aurelien-brabant/ft_transcendance/pull/120/commits/8c765edd88e971d03881e524c9c94aeb4957e56a).